### PR TITLE
Track 2 Mgmt Preview docs and readme update based on new namespace

### PR DESCRIPTION
**Pull request by nickzhums**

### CODE REVIEW

File: sdk/eventhub/Azure.ResourceManager.EventHubs/README.md [hunk 1]
```diff
@@ -1,69 +1,51 @@
-# README.md template
+# Azure Management EventHubs client library for .NET
```
Comment (jsquire) [inline sdk/eventhub/Azure.ResourceManager.EventHubs/README.md:1]: nit:  There should be a space in the product name:  `Event Hubs`.    Leaving one client here, but the same is true for the other non-code references below.

File: sdk/eventhub/Azure.ResourceManager.EventHubs/README.md [hunk 2]
```diff
@@ -1,69 +1,51 @@
...
-### Authenticate the client
+```PowerShell
+Install-Package Azure.ResourceManager.EventHubs
+```
 
-If your library requires authentication for use, such as for Azure services, include instructions and example code needed for initializing and authenticating.
+## Documentation
 
-For example, include details on obtaining an account key and endpoint URI, setting environment variables for each, and initializing the client object.
+Various documentation is available to help you learn how to use this package
```
Comment (jsquire) [inline sdk/eventhub/Azure.ResourceManager.EventHubs/README.md:19]: This reads a bit awkwardly.   Perhaps something like `various forms of documentation`   or  `various resources` ?

File: sdk/eventhub/Azure.ResourceManager.EventHubs/README.md [hunk 3]
```diff
@@ -1,69 +1,51 @@
...
-## Next steps
-
-* Provide a link to additional code examples, ideally to those sitting alongside the README in the package's `/samples` directory.
-* If appropriate, point users to other packages that might be useful.
-* If you think there's a good chance that developers might stumble across your package in error (because they're searching for specific functionality and mistakenly think the package provides that functionality), point them to the packages they might be looking for.
+-   File an issue via [Github
+    Issues](https://github.com/Azure/azure-sdk-for-net/issues)
+-   Check [previous
+    questions](https://stackoverflow.com/questions/tagged/azure+.net)
+    or ask new ones on StackOverflow using azure and .NET tags.
```
Comment (jsquire) [inline sdk/eventhub/Azure.ResourceManager.EventHubs/README.md:31]: nit:  "... using the `azure` and `.net` tags."   _(Stack Overflow tags are lower case)_

Comment (jsquire) [inline sdk/eventhub/Azure.ResourceManager.EventHubs/README.md:31]: nit:  "Stack Overflow"   _(missing space)_

File: sdk/eventhub/Azure.ResourceManager.EventHubs/README.md [hunk 4]
```diff
@@ -1,69 +1,51 @@
...
+To create an authenticated client and start interacting with Azure resources, please see the [quickstart guide here](https://github.com/Azure/azure-sdk-for-net/blob/master/doc/mgmt_preview_quickstart.md)
 
 ### Install the package
 
-First, provide instruction for obtaining and installing the package or library. This section might include only a single line of code, like `pip install package-name`, but should enable a developer to successfully install the package from NuGet, pip, npm, Maven, or even cloning a GitHub repository.
-
-### Prerequisites
-
-* You must have an [Azure subscription](https://azure.microsoft.com/free/)
+Install the Azure Management EventHubs client library for .NET with [NuGet](https://www.nuget.org/):
```
Comment (jsquire) [inline sdk/eventhub/Azure.ResourceManager.EventHubs/README.md:11]: Are we calling this a "client library?"    My understanding was that we used the term "client" only with respect to the data plane.

Comment (jsquire) [reply sdk/eventhub/Azure.ResourceManager.EventHubs/README.md:11]: @AlexGhiondea, @KrzysztofCwalina:  Would you be so kind as to offer your thoughts on the above?

**Review by jsquire — DISMISSED** [review]

File: sdk/keyvault/Azure.ResourceManager.KeyVault/README.md [hunk 5]
```diff
@@ -1,69 +1,51 @@
...
-First, provide instruction for obtaining and installing the package or library. This section might include only a single line of code, like `pip install package-name`, but should enable a developer to successfully install the package from NuGet, pip, npm, Maven, or even cloning a GitHub repository.
-
-### Prerequisites
-
-* You must have an [Azure subscription](https://azure.microsoft.com/free/)
+Install the Azure Management KeyVault client library for .NET with [NuGet](https://www.nuget.org/):
 
-### Authenticate the client
+```PowerShell
+Install-Package Azure.ResourceManager.KeyVault
```
Comment (heaths) [inline sdk/keyvault/Azure.ResourceManager.KeyVault/README.md:14]: Aren't these packages still previews? This needs to include the `--version <preview version number>` and changed with each release like we do for the data-plane SDKs.

File: sdk/keyvault/Azure.ResourceManager.KeyVault/README.md [hunk 6]
```diff
@@ -1,69 +1,51 @@
...
+```
 
-If your library requires authentication for use, such as for Azure services, include instructions and example code needed for initializing and authenticating.
+## Documentation
 
-For example, include details on obtaining an account key and endpoint URI, setting environment variables for each, and initializing the client object.
+Various documentation is available to help you learn how to use this package
```
Comment (heaths) [inline sdk/keyvault/Azure.ResourceManager.KeyVault/README.md:19]: "documentation" is generally plural, so "Various documentation" doesn't read right. I'd just say "Documentation ..." and the list below works better.

File: sdk/keyvault/Azure.ResourceManager.KeyVault/README.md [hunk 7]
```diff
@@ -1,69 +1,51 @@
...
+When you submit a pull request, a CLA-bot will automatically determine
+whether you need to provide a CLA and decorate the PR appropriately
+(e.g., label, comment). Simply follow the instructions provided by the
+bot. You will only need to do this once across all repositories using
+our CLA.
 
-![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-net%2Fsdk%2Ftemplate%2FAzure.Template%2FREADME.png)
+This project has adopted the Microsoft Open Source Code of Conduct. For
+more information see the Code of Conduct FAQ or contact
+<opencode@microsoft.com> with any additional questions or comments.
```
Comment (heaths) [inline sdk/keyvault/Azure.ResourceManager.KeyVault/README.md:51]: All these READMEs should probably have the impressions image we use for stats. See data-plane READMEs for examples.

File: sdk/eventhub/Azure.ResourceManager.EventHubs/README.md (hunk 1 above)
Comment (nickzhums) [reply sdk/eventhub/Azure.ResourceManager.EventHubs/README.md:1]: Got it, thanks

File: sdk/eventhub/Azure.ResourceManager.EventHubs/README.md (hunk 2 above)
Comment (nickzhums) [reply sdk/eventhub/Azure.ResourceManager.EventHubs/README.md:19]: Yep

File: sdk/eventhub/Azure.ResourceManager.EventHubs/README.md (hunk 3 above)
Comment (nickzhums) [reply sdk/eventhub/Azure.ResourceManager.EventHubs/README.md:31]: good eye :=)

File: sdk/keyvault/Azure.ResourceManager.KeyVault/README.md (hunk 7 above)
Comment (nickzhums) [reply sdk/keyvault/Azure.ResourceManager.KeyVault/README.md:51]: not all management RPs currently have data-plane counterpart, is there an impression image generation tool ?

File: doc/mgmt_preview_quickstart.md [hunk 8]
```diff
@@ -0,0 +1,338 @@
...
+following:
+```csharp
+    using Azure.Identity;
+    using Azure.ResourceManager.Resources;
+    using Azure.ResourceManager.Resources.Models;
+    using System;
+    ...
+    var subscriptionId = Environment.GetEnvironmentVariable("AZURE_SUBSCRIPTION_ID");
+    var resourceClient = new ResourcesManagementClient(subscriptionId, new DefaultAzureCredential());
+    var resourceGroupsClient = resourceClient.GetResourceGroupsClient();
```
Comment (erich-wang) [inline doc/mgmt_preview_quickstart.md:94]:
```suggestion
    var resourceGroupsClient = resourceClient.ResourceGroups;
```

File: doc/mgmt_preview_quickstart.md [hunk 9]
```diff
@@ -0,0 +1,338 @@
...
+```
+From this code snippet, we showed that in order to interact with Resources, we need to create a top-level client first (**ResourcesManagementClient**), then get the corresponding sub-resource client we are interested in, in this case we called **.GetResourceGroupsClient()** to get a ResourceGroupsClient
```
Comment (erich-wang) [inline doc/mgmt_preview_quickstart.md:96]:
```suggestion
From this code snippet, we showed that in order to interact with Resources, we need to create a top-level client first (**ResourcesManagementClient**), then get the corresponding sub-resource client we are interested in, in this case we called **.ResourceGroups** to get a ResourceGroupsOperations
```

File: doc/mgmt_preview_quickstart.md [hunk 10]
```diff
@@ -0,0 +1,338 @@
...
+            var computeClient = new ComputeManagementClient(subscriptionId, new DefaultAzureCredential());
+            var networkClient = new NetworkManagementClient(subscriptionId, new DefaultAzureCredential());
+            var resourcesClient = new ResourcesManagementClient(subscriptionId, new DefaultAzureCredential());
+ 
+            var virtualNetworksClient = networkClient.GetVirtualNetworksClient();
+            var networkInterfaceClient = networkClient.GetNetworkInterfacesClient();
+            var publicIpAddressClient = networkClient.GetPublicIPAddresssesClient();
+            var availabilitySetsClient = computeClient.GetAvailabilitySetsClient();
+            var virtualMachinesClient = computeClient.GetVirtualMachinesClient();
+            var resourceGroupClient = resourcesClient..GetResourceGroupsClient();
```
Comment (erich-wang) [inline doc/mgmt_preview_quickstart.md:188]:
```suggestion
            var virtualNetworksClient = networkClient.VirtualNetworks;
            var networkInterfaceClient = networkClient.NetworkInterfaces;
            var publicIpAddressClient = networkClient.PublicIPAddressses;
            var availabilitySetsClient = computeClient.AvailabilitySets;
            var virtualMachinesClient = computeClient.VirtualMachines;
            var resourceGroupClient = resourcesClient.ResourceGroups;
```

**Review by erich-wang — APPROVED** [review]

**Pull request merged**